apps/frontend/src/hooks/experiment/useExperiment.ts

```diff
@@ -22,12 +22,13 @@ export function useExperiment(experimentPk: number) {
         if (experiment) {
           setExperiment(experiment);
         }
+        setLoading(false);
```

Comment:
Will we be able to reach here if the call fails completely maybe it safe to put it in finally.